packages/oas-extensions/tsconfig.json

```diff
@@ -1,14 +1,8 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "allowJs": true,
     "baseUrl": "./src/",
```

Comment:
it looks like all of these have the same `baseUrl`, should we move that to the base config?

Reply:
Moving these to the base config makes TS think that our base URL for the source is `./src` not `packages/*/src`.

Reply:
huh, I got it working here: https://github.com/readmeio/api/pull/733/files#diff-0b280a445be167f54cd916c0718c952b8e0d4ca9621903d05eec25efd4c6ed6d